contracts/test/Teleporter/AvalancheICTTRouterFixedFees/AvalancheICTTRouterFixedFeesErc20TokenTest.t.sol

```diff
@@ -103,6 +116,37 @@ contract AvalancheICTTRouterFixedFeesErc20TokenTest is Test {
         _;
     }
 
+    function testRevertsIfBridgedAmountNotEnoughToPayMinBridgeFees()
+        public
+        registerTokenBridge
+        fundBridgerAccount
+    {
+        uint256 tooLittleAmount = 0.000000001 ether;
+
+        uint256 primaryRelayerFee = (tooLittleAmount * primaryRelayerFeeBips) / 10_000;
+        MinBridgeFees memory minBridgeFees =
+            MinBridgeFees(minPrimaryRelayerFee, minSecondaryRelayerFee);
+
+        erc20Token.approve(address(tokenBridgeRouter), tooLittleAmount);
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                IAvalancheICTTRouterFixedFees
+                    .AvalancheICTTRouterFixedFees__RelayerFeesTooLow
+                    .selector,
+                primaryRelayerFee,
+                0,
```

Comment:
Not sure why this is `0` if the `minSecondaryRelayerFee` is `> 0`.

Reply:
This is not `minSecondaryRelayerFee`, this is the `secondaryFeeAmount` which is set to `0` because it is not a `multihop`. The parameters of the error are: `primaryFeeAmount`, `secondaryFeeAmount` and `MinBridgeFees` (which contains both `minPrimaryRelayerFee` and `minSecondaryRelayerFee`).